client/features/auth-pages/BackOffice.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState, useEffect } from 'react';
+import { Box, Button } from '@mui/material';
+import DatePicker from 'react-datepicker';
+import style from '@styles/BackOffice.module.css';
+
+import 'react-datepicker/dist/react-datepicker.css';
+
+const BackOffice = () => {
+  const today = new Date();
+  const tomorrowMidnight = new Date(today);
+  tomorrowMidnight.setDate(today.getDate() + 1);
+  const [endDate, setEndDate] = useState(
+    new Date(new Date(tomorrowMidnight).setHours(0, 0, 0, 0)),
+  );
+  const [startDate, setStartDate] = useState(
+    new Date(new Date().setHours(0, 0, 0, 0)),
+  );
+
+  const startDateMs = startDate.getTime();
+  const endDateMs = endDate.getTime();
+
+  // Calculate the difference in milliseconds
+  const differenceMs = Math.abs(endDateMs - startDateMs); // Use Math.abs to get a positive difference
+  // Convert milliseconds to days
+  const millisecondsPerDay = 1000 * 60 * 60 * 24;
+  const differenceDays = Math.floor(differenceMs / millisecondsPerDay);
```

Comment:
Since you are using `differenceDays` in both the `useEffect` and `downloadLogs`, consider making a util function to improve the readability:

```typescript
const differenceDays = useMemo(() => {
    const startDateMs = startDate.getTime();
    const endDateMs = endDate.getTime();
    const differenceMs = Math.abs(endDateMs - startDateMs);
    return Math.floor(differenceMs / millisecondsPerDay);
  }, [startDate, endDate]);
```

Reply:
Done.